ethcore/engines/authority-round/src/lib.rs

```diff
@@ -720,17 +720,22 @@ fn header_signature(header: &Header, empty_steps_transition: u64) -> Result<Sign
 	.as_val::<H520>().map(Into::into)
 }
 
-// extracts the raw empty steps vec from the header seal. should only be called when there are 3 fields in the seal
-// (i.e. header.number() >= self.empty_steps_transition)
-fn header_empty_steps_raw(header: &Header) -> &[u8] {
-	header.seal().get(2).expect("was checked with verify_block_basic; has 3 fields; qed")
+// Extracts the empty steps from the header seal. Returns data only when there are 3 fields in the seal
```

Comment:
Why remove `raw` from the docs? Do you think it's redundant since the method name (and types) indicate it's raw?

Reply:
I removed it because I didn't think "raw" made sense here. In what sense are the steps "raw"? Maybe it should be "Extracts the RLP of the empty steps from the header seal. …"?

Reply:
`s/raw/RLP bytes/` makes sense!